Documentation/Contributing/TestUtils.md

```diff
@@ -0,0 +1,34 @@
+# Adobe Experience Platform Test Utilities (AEPTestUtils) contribution guide
+
+This guide describes the guidelines for contributing to the `AEPTestUtils` library.
+
+Generally, the APIs exposed by `AEPTestUtils` should be considered public APIs and treated with the same level of care in terms of maintenance commitment and breaking changes.
+
+The library is not published as part of Core, as it is internal to Mobile SDK development. Instead, it uses a git tag in the format `testutils-x.y.z`, where `x.y.z` generally follows semantic versioning rules, and its major version matches that of Core. This way, `AEPTestUtils` can be updated independently from Core.
```

Comment:
Can you add "This pod isn't available on CocoaPods and can only be referenced using its GitHub URL" and also a sample that can be copy/pasted in Podfile.